signup-service/src/main/java/io/mosip/signup/services/RegistrationService.java

```diff
@@ -301,7 +301,10 @@ public RegistrationStatusResponse updatePassword(ResetPasswordRequest resetPassw
         restRequest.setRequesttime(IdentityProviderUtil.getUTCDateTime());
         restRequest.setRequest(identityRequest);
 
-        log.debug("Transaction {} : start reset password", transactionId);
+        return restRequest;
+    }
+
+    private void validateResetPasswordRequest(RestRequestWrapper restRequest, String transactionId ) {
```

Comment:
The method name is validateResetPasswordRequest but the method is only sending a patch request to the ID Repo server. We should rename the method.